ui/src/dashboards/selectors/index.ts

```diff
@@ -50,19 +50,13 @@ export const sortDashboardByName = (dashboards: Dashboard[]): Dashboard[] =>
 // 10-11:00am local time (offset depending on timezone)
 export const setTimeToUTC = (date: string): string => {
   const offset = new Date(date).getTimezoneOffset()
-  if (offset > 0) {
-    return moment
-      .utc(date)
-      .subtract(offset, 'minutes')
-      .format()
+  if (offset === 0) {
+    return moment.utc(date).format()
```

Comment:
Is this needed? if offset = 0, would `.subtract(offset, 'minutes')` not just return zero? I don't know the answer just thought I'd check

Reply:
That's a great question / point. I think I had originally implemented this so that it wouldn't overwrite formatted data unnecessarily. Let me just check real quick to see if it makes a difference

Reply:
Great point, totally didn't consider this. You're 100% correct